js/app.js

```diff
@@ -0,0 +1,60 @@
+
+/*fetchCountries = async () => {
+  const response = await fetch('https://restcountries.com/v2/all');
+  const myJson = await response.json(); //extract JSON from the http response
+  console.log(myJson);
+  localStorage['jsonData'] = JSON.stringify(myJson);*/
+
+
+
+
+
+const flags = document.getElementById('flags')
+
+document.addEventListener("DOMContentLoaded", e => {
+  fetchData()
+})
+
+const fetchData = async () => {
+  try {
+    const res = await fetch('https://restcountries.com/v2/all')
+    //const res = await fetch('https://restcountries.com/v3.1/all') no jala con la versiòn 3.1, no sè por qué? será la sintaxis?
```

Comment:
That's because the objects response is different from `v2` and `v3`, I can explain why this happens in today's meeting.